script.js

```diff
@@ -0,0 +1,15 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('#mobile-navbar');
+const mobileMenuList = document.querySelector('#mobile-menu-list');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  mobileMenuList.classList.toggle('hide-mobile-menu');
+});
+
+document.querySelectorAll('.hide-mobile-menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  mobileMenuList.classList.remove('hide-mobile-menu');
+}));
```

Comment:
- [ OPTIONAL ] This issue mentioned by the previous reviewer still exists, it is optional since it wasn't asked before but I would highly recommend you doing it